### DISCUSSION TITLE
Account  Suspended

### DISCUSSION THREAD

**Discussion by MohamedLasheen22** [community/community#102921] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

on 1/31/2024, My github account has been suspended and I am reaching out to support via email but there is no response.
this message appears "Access to your account has been suspended due to a violation of our [Terms of Service].
 Please [contact support] for more information."
The github account has everything which I have achieved in tech till now.

Deleted account username - mohamedlasheen92
Ticket ID - 2565123

I can't lose the account. I request anyone to please suggest what should I do in this case?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @MohamedLasheen22! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, be advised that the community is unable to assist or escalate in these matters. An important part of the restoration process is for security purposes, we can discuss account details only when we have established that we are in contact with the account's legitimate owner. To do this, we must receive communication from a current, verified email address associated with the account in question. I see that your ticket was submitted yesterday, so you should receive a response soon. 

I've gone ahead and moved this to the correct category for you. Good luck!
